scripts/linters/pylint_extensions_test.py

```diff
@@ -2977,3 +2977,48 @@ def some_random_function(param1, param2):
 
         with self.checker_test_object.assertNoMessages():
             self.checker_test_object.checker.visit_functiondef(def_node)
+
+
+class ConcatenationCheckerTests(unittest.TestCase):
+
+    def setUp(self):
+        super(ConcatenationCheckerTests, self).setUp()
+        self.checker_test_object = testutils.CheckerTestCase()
+        self.checker_test_object.CHECKER_CLASS = (
+            pylint_extensions.ConcatenationChecker)
+        self.checker_test_object.setup_method()
+
+    def test_string_concatenation(self):
```

Comment:
There should be a test method for each kind of case - like here atleast two are needed - one for the case where error is raised, and other where no error should be raised. Also we follow a standard convention for naming the test methods - you can find info about both these things in our [guide](https://github.com/oppia/oppia/wiki/Writing-backend-tests#guidelines-for-writing-good-tests) to help you structure the tests in a better way.

Reply:
Done